heron/executor/src/python/heron_executor.py

```diff
@@ -585,6 +606,44 @@ def _get_streaming_processes(self):
 
     return retval
 
+  def _get_ckptmgr_process(self):
+    ''' Get the command to start the checkpoint manager process'''
+
+    ckptmgr_main_class = 'com.twitter.heron.ckptmgr.CheckpointManager'
+
+    ckptmgr_cmd = [os.path.join(self.heron_java_home, "bin/java"),
+                   '-Xmx512M',
```

Comment:
Since the ckptmgr will not be started for now, and `metricsmgr` and `metricscacgemgr` are having the same problem. How about we solve the problem in another separate PR after this is merged?